android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/SpinnerFactoryTest.java

```diff
@@ -1,72 +1,68 @@
 package com.vijay.jsonwizard.widgets;
 
 import android.util.Pair;
+import android.view.View;
+import android.widget.RelativeLayout;
 
 import com.vijay.jsonwizard.BaseTest;
+import com.vijay.jsonwizard.R;
+import com.vijay.jsonwizard.activities.JsonFormActivity;
 import com.vijay.jsonwizard.constants.JsonFormConstants;
+import com.vijay.jsonwizard.customviews.MaterialSpinner;
+import com.vijay.jsonwizard.fragments.JsonFormFragment;
+import com.vijay.jsonwizard.interfaces.CommonListener;
 
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
 
-import static org.junit.Assert.assertEquals;
+import java.util.List;
+import java.util.Set;
 
 /**
  * Created by Vincent Karuri on 03/04/2020
  */
 public class SpinnerFactoryTest extends BaseTest {
```

Comment:
extend `FactoryTest` instead, remove the `JsonFormActivity` declaration and assignment call super on `setUp`